replica-healthcheck 0.3.8 fails to start

**Describe the bug**

replica-healthcheck 0.3.8 from docker image fails to start with:

```
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '@eth-optimism/sdk'
Require stack:
- /opt/optimism/packages/replica-healthcheck/dist/healthcheck-server.js
- /opt/optimism/packages/replica-healthcheck/dist/index.js
- /opt/optimism/packages/replica-healthcheck/dist/exec/run-healthcheck-server.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/opt/optimism/packages/replica-healthcheck/dist/healthcheck-server.js:13:15)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/opt/optimism/packages/replica-healthcheck/dist/healthcheck-server.js',
    '/opt/optimism/packages/replica-healthcheck/dist/index.js',
    '/opt/optimism/packages/replica-healthcheck/dist/exec/run-healthcheck-server.js'
  ]
}
```

--- a/.changeset/big-cycles-listen.md
+++ b/.changeset/big-cycles-listen.md
@@ -1,0 +1,5 @@
+---
+'@eth-optimism/replica-healthcheck': patch
+---
+
+Fix bug in replica healthcheck dockerfile

--- a/ops/docker/Dockerfile.replica-healthcheck
+++ b/ops/docker/Dockerfile.replica-healthcheck
@@ -12,6 +12,8 @@ COPY --from=builder /optimism/yarn.lock .
 COPY --from=builder /optimism/node_modules ./node_modules
 
 # copy deps (would have been nice if docker followed the symlinks required)
+COPY --from=builder /optimism/packages/sdk/package.json ./packages/sdk/package.json
+COPY --from=builder /optimism/packages/sdk/dist ./packages/sdk/dist
 COPY --from=builder /optimism/packages/core-utils/package.json ./packages/core-utils/package.json
 COPY --from=builder /optimism/packages/core-utils/dist ./packages/core-utils/dist
 COPY --from=builder /optimism/packages/common-ts/package.json ./packages/common-ts/package.json

